app/helpers/anchor/form_builder.rb

```diff
@@ -0,0 +1,122 @@
+module Anchor
+  class FormBuilder < ActionView::Helpers::FormBuilder
+    delegate :concat, :content_tag, :render, to: :@template
+
+    def error_message_for(attribute, options = {})
+      render ErrorMessageComponent.new(
+        form_builder: self,
+        attribute:,
+        **options
+      )
+    end
+
+    def label(attribute, text = nil, options = {}, &)
+      render LabelComponent.new(
+        form_builder: self,
+        attribute:,
+        text:,
+        **options
+      ) do |component|
+        component.with_label_tag do
+          super attribute, text, options.merge(component.options), &
+        end
+      end
+    end
+
+    def text_field(attribute, options = {})
+      render TextFieldComponent.new(
+        form_builder: self,
+        attribute:
+      ) do |component|
+        component.with_input_tag do
+          super attribute, options.merge(component.options)
+        end
+      end
+    end
+
+    # The following methods are pending to be replaced with ViewComponents.
+
+    def collection_radio_buttons(
+      attribute,
+      collection,
+      value_method,
+      text_method,
+      options = {},
+      html_options = {}
+    )
+      content_tag :div, class: "flex flex-col space-y-2 text-base" do
+        super(
+          attribute,
+          collection,
+          value_method,
+          text_method,
+          options,
+          html_options.merge(class: TailwindConstants::RADIO)
+        ) do |radio|
+          content_tag :div, class: "inline-flex items-center gap-1" do
+            concat(radio.radio_button(
+              data: {
+                testId: "radio-#{attribute}-#{radio.value}".gsub("_", "-"),
```

Comment:
Can this instead use `dasherize`?

```suggestion
                testId: "radio-#{attribute}-#{radio.value}".dasherize,
```

Reply:
Yes, I can change it, though this was just copied from the existing file on Infirmary. I was trying to minimize changes outside the scope of this PR.